### PR TITLE
Add CreateOrUpdateRepoCustomPropertyValues

### PR DESCRIPTION
**Pull request by ganeshkumarsv**
Adds method CreateOrUpdateRepoCustomPropertyValues

### CODE REVIEW

**Comment by gmlewis** [comment]
Thank you, @ganeshkumarsv !
I saw the commit titled "no idea what im doing" and thought I might help out...  In CONTRIBUTING.md, we have some helpful tips that should assist you.  Please let me know if you have any question, but it looks like you are on the right track.  You can also look at some of the other files for examples on code coverage.

**Comment by ganeshkumarsv** [comment]
> Thank you, @ganeshkumarsv ! I saw the commit titled "no idea what im doing" and thought I might help out... In CONTRIBUTING.md, we have some helpful tips that should assist you. Please let me know if you have any question, but it looks like you are on the right track. You can also look at some of the other files for examples on code coverage.

Thanks for commenting @gmlewis. Running metadata.sh makes modification for a lot of URLs and erroring out. I don't see a way to update openapi configs only the one endpoint i'm updating. Any help appreciated. Thank you 🙇

**Comment by gmlewis** [comment]
> Thanks for commenting @gmlewis. Running metadata.sh makes modification for a lot of URLs and erroring out. I don't see a way to update openapi configs only the one endpoint i'm updating. Any help appreciated. Thank you 🙇

You shouldn't have to deal with the metadata or OpenAPI stuff... you should only have to make sure your formatting has been fixed and the generator has been run, and you can revert any changes made to the metadata.

**Comment by ganeshkumarsv** [comment]
I tried running `go run github/gen-accessors.go` but it is not generating anything. Am i missing something?

**Comment by ganeshkumarsv** [comment]
I removed my changes from `openapi_operations.yaml` as u mentioned. The linter fails with the error `could not find operation "PATCH /repos/{owner}/{repo}/properties/values" in openapi_operations.yaml` 

Can I know how i can fix this error?

**Comment by gmlewis** [comment]
@ganeshkumarsv - now that #3104 is merged, please try to update this branch using `git merge master` and see if that solves the problems you were seeing before.

**Comment by ganeshkumarsv** [comment]
Thanks @gmlewis. I updated the branch and it fixed the linter. All the checks are passing now.

File: github/repos_properties.go
Comment (gmlewis) [inline github/repos_properties.go:46]:
```suggestion
		Properties []*CustomPropertyValue `json:"properties"`
	}{
		Properties: customPropertyValues,
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @ganeshkumarsv !
Just one minor tweak, please, then we should be ready for a second LGTM+Approval from any other contributor to this repo before merging.

**Review by gmlewis — APPROVED** [review]
Thank you, @ganeshkumarsv !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Comment by ganeshkumarsv** [comment]
@liaodaniel I need one more approval to merge this PR. Can you please review? 🙇

**Review by AbbanMustafa — APPROVED** [review]

**Comment by ganeshkumarsv** [comment]
@gmlewis Thanks for merging. Is it possible to create a new release?

**Comment by gmlewis** [comment]
> @gmlewis Thanks for merging. Is it possible to create a new release?

This is now released in v61: https://github.com/google/go-github/releases/tag/v61.0.0

**Pull request merged**
